jsonnet/jsonnetfile.json

```diff
@@ -0,0 +1,15 @@
+{
+  "version": 1,
+  "dependencies": [
+    {
+      "source": {
+        "git": {
+          "remote": "https://github.com/kubernetes-monitoring/kubernetes-mixin.git",
+          "subdir": ""
+        }
+      },
+      "version": "master"
```

Comment:
Ideally, this should be pinned to release branches before OCP code freeze. https://github.com/kubernetes-monitoring/kubernetes-mixin#releases. release-0.10 is tied to Kubernetes 1.23 and OCP 4.10